asf_hp_cost_estimator_model/pipeline/model_evaluation/cross_validation.py

```diff
@@ -58,159 +49,171 @@ def argparse_setup():
     return parser.parse_args()
 
 
-if __name__ == "__main__":
-    args = argparse_setup()
-    lower_quantile = args.lower_quantile
-    upper_quantile = args.upper_quantile
+def create_model(
+    quantile: float, model_params: Dict[str, Any], random_state: int
```

Comment:
`random_state` slightly redundant given this is just passed originally from your config, could just remove and use the config internally.